test/loraxtest.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python3
+
+from cockpittest import TestCase
+
+class LoraxTestCase(TestCase):
+    def runLoraxTest(self, script=""):
+        extra_env = []
+        if self.sit:
+            extra_env.append("COMPOSER_TEST_FAIL_FAST=1")
+
+        r = self.execute(["CLI=/usr/sbin/lorax",
+                          "TEST=" + self.id(),
+                          "PACKAGE=lorax",
+                          *extra_env,
+                          "/tests/test_lorax.sh", script])
+        self.assertEqual(r.returncode, 0)
```

Comment:
This is very similar to `runCliTest`. Both are running a beakerlib script on the test VM, just with different names. Can we implement both with the same function? (This only applies if we don't change approach as I proposed above.)

Reply:
Sure, just need to pass in a few more args.